packages/editor-ui/src/components/canvas/elements/edges/CanvasEdge.vue

```diff
@@ -36,5 +80,25 @@ const path = computed(() =>
 		:label-bg-style="{ fill: 'red' }"
 		:label-bg-padding="[2, 4]"
 		:label-bg-border-radius="2"
+		tabindex="2"
```

Comment:
What's the reason for manually setting `tabindex` here and in `CanvasNode`? Since tab should toggle node-creator, we wouldn't expect for the tab to select individual nodes/edge anyway, right?

Reply:
The reason is that `@keydown` doesn't work on divs without `tabindex`. Since I've refactored the keydown handler, this won't be needed anymore.